gamemodes/terrortown/entities/entities/ttt_hat_deerstalker.lua

```diff
@@ -65,26 +88,17 @@ if SERVER then
 		self:SetParent(nil)
 
 		self:SetBeingWorn(false)
-		self:SetUseType(SIMPLE_USE)
 
 		-- only now physics this entity
 		self:PhysicsInit(SOLID_VPHYSICS)
 		self:SetSolid(SOLID_VPHYSICS)
 		self:SetMoveType(MOVETYPE_VPHYSICS)
 
-		-- position at head
+		-- if we're not already on the player's head,
 		if IsValid(ply) then
-			local bone = ply:LookupBone("ValveBiped.Bip01_Head1")
-			if bone then
-				local pos, ang = ply:GetBonePosition(bone)
-				self:SetPos(pos)
-				self:SetAngles(ang)
-			else
-				local pos = ply:GetPos()
-				pos.z = pos.z + 68
-
-				self:SetPos(pos)
-			end
+			local pos, ang = ply:GetBonePosition(bone)
+			self:SetPos(pos)
+			self:SetAngles(ang)
```

Comment:
This is wrong, bone is not defined here.

Reply:
good catch, i clearly meant to factor out as much of the bone lookup as possible and missed this